apps/ex_wire/lib/ex_wire/p2p/manager.ex

```diff
@@ -21,28 +21,28 @@ defmodule ExWire.P2P.Manager do
   and a `peer` to be provided. This function starts the encrypted handshake with
   the `peer`.
   """
-  @spec new_outbound_connection(ExWire.TCP.socket(), Peer.t()) :: Connection.t()
-  def new_outbound_connection(socket, peer) do
+  @spec new_outbound_connection(Connection.t()) :: Connection.t()
+  def new_outbound_connection(state) do
```

Comment:
`conn` or something related to Connection would be a better name than `state` here and below.